### PR TITLE
`slackr_ims` was not correctly returning the channel IDs.

### PR DESCRIPTION
**Pull request by jcrodriguez1989**
`slackr_ims` was `left_join`ing `users.list` and `conversations.list` by `id`, but `conversations.list`'s `id` is the ID of the channel.
I think the correct way would be to join `users.list`'s `id` with `conversations.list`'s `user`.

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -25,10 +25,12 @@ export(slackr_users)
 export(with_pagination)
 importFrom(cachem,cache_disk)
 importFrom(cachem,cache_mem)
+importFrom(dplyr,`%>%`)
```
Comment (mrkaye97) [inline NAMESPACE]: The pipe is already imported from `magrittr`, so you don't need to re-import it from `dplyr`.

With that said though, I'd actually prefer to clean this up a bit with `usethis::use_pipe()`. I'm happy to make that change, but if you wanted to drive-by fix it as part of this PR, that'd be great!

File: R/slackr_utils.R [hunk 2]
```diff
@@ -29,21 +29,22 @@ slackr_channels <- function(token = Sys.getenv("SLACK_TOKEN")) {
 #' Get a data frame of Slack IM ids
 #'
 #' @param token Authentication token bearing required scopes.
-#' @importFrom dplyr left_join
+#' @importFrom dplyr `%>%` left_join rename
```
Comment (mrkaye97) [inline R/slackr_utils.R]: See comment above about the pipe

**Review by mrkaye97 — COMMENTED** [review]
Thanks for the contribution @jcrodriguez1989! This looks good AFAICT, but I'll need to figure out how to get `CMD check` to run on this PR before merging. I just tried updating the CI config. Do you mind rebasing off `master` to get up to date with that change? I'm not sure if it'll work, but we'll see. 

It'd also be great if you could bump the package version to `3.1.1`. I see this is a patch bug fix

File: NAMESPACE (hunk 1 above)
Comment (jcrodriguez1989) [reply NAMESPACE]: I've used `usethis::use_pipe()` and removed pipe imports (`@importFrom ... %>%`).

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# slackr 3.1.1
+
+* `slackr_ims` bug fix.
+* Using `usethis::use_pipe()` to import `{magrittr}`'s pipe.
+
```
Comment (mrkaye97) [inline NEWS.md:5]: Thanks for adding this!

File: R/call_slack_api.R [hunk 4]
```diff
@@ -222,7 +222,6 @@ with_pagination <- function(fun, extract) {
 #'
 #' @references https://api.slack.com/methods/auth.test
 #' @export
-#' @importFrom magrittr %>%
```
Comment (mrkaye97) [inline R/call_slack_api.R:225]: :+1: Thanks for DRY here

**Review by mrkaye97 — APPROVED** [review]
Nice, this looks great! Thanks for the contribution @jcrodriguez1989! 🎉
